recipes/Tedlium2/ASR/transformer/hparams/branchformer_large.yaml

```diff
@@ -0,0 +1,259 @@
+# ############################################################################
+# Model: E2E ASR with Transformer
+# Encoder: Branchformer Encoder
+# Decoder: Transformer Decoder + (CTC/ATT joint) beamsearch
+# Tokens: unigram
+# losses: CTC + KLdiv (Label Smoothing loss)
+# Training: Tedlium2
+# Authors:  Titouan Parcollet, Shucong Zhang
+# ############################################################################
+# Seed needs to be set at top of yaml, before objects with parameters are made
+
+seed: 3407 
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref results/branchformer_large/<seed>
+output_wer_folder: !ref <output_folder>/
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+
+# Data files
+# NOTE: This script will NOT do data preparation, since the data preparation 
```

Comment:
it's a bit weird. What is the exact reason to avoid doing data preparation inside the train.py file? We are doing that for all our recipes. Furthermore, what happens if tmrw, we release a ckpt with the tokenizer? The end user will have to run the Tokenizer train.py recipe to generate the CSV files which is a bit weird right?